### DISCUSSION TITLE
Project recover?

### DISCUSSION THREAD

**Discussion by rupkatAI** [photopea/photopea#6270] [General]
Is there any chance to recover autsaved .psd cause somehow theres one missing project on my folder where i always save all my projects

**Comment by photopea** [comment]
If you are working in Photopea and it crashes (e.g. your battery dies), you should see your work recovered the next time you open Photopea.

But if you close Photopea and you confirm that the unsaved work will be lost, your work will be lost :(

Note, that there exist browser extensions, which break our backup mechanism, or break the correct download of files from Photopea.

![image (1)](https://github.com/photopea/photopea/assets/15955558/facbb052-b480-4e8f-a6c6-16787225cfee)

**Reply by rupkatAI** [reply]
Thanks for reply, everything is now working fine :)

**Reply by paloha** [reply]
Hello, I'd like to report that today my computer crashed (I run out of memory and lost my Gnome session). After I restarted my browser and opened the tab with Photopea, all my unsaved projects loaded up properly. However, one project - the only one I actually previously saved as PSD and then worked on it for a few hours (without saving), did not load. So now, I only have my old version. Does Photopea not restore projects that were saved at some point in their lifetime? If this is the case, I believe better would be to rather restore all.

**Comment by photopea** [comment]
The backup mechanism of Photopea is described here: https://www.photopea.com/tuts/photopea-opens-up-old-files/

If you use any browser extesions, which have your permission to change the behaviour of Photopea, they could break our backup mechanism.

**Reply by paloha** [reply]
Hello, I am aware of this. But all my projects loaded as expected, except one - which was weird and felt like a bug, that is why I wanted to report this. If it was an extension issue, it would most probably interfere with loading all the projects. In any case, I should save my work more often.
